### PR TITLE
Remove duplicate code

### PR DESCRIPTION
**Pull request by MichalOkon**
In the introductory part of the book there was a code duplication in the example presenting Beforeeach annotation.

### CODE REVIEW

File: chapters/getting-started/test-automation.md [hunk 1]
```diff
@@ -197,23 +197,20 @@ class RomanNumeralTest {
 
   @Test
   void convertSingleDigit() {
-    roman = new RomanNumeral();
```
Comment (mauricioaniche) [inline chapters/getting-started/test-automation.md:200]: Great catch

Can you remove the empty lines in between? Making it shorter will also help the future PDF version of the book.

File: chapters/pragmatic-testing/test-code-quality.md [hunk 2]
```diff
@@ -649,11 +648,11 @@ void taxesForCompanies() {
 } 
 ```
 
-Note how our `InvoiceBuilder` (which we show the code soon) clearly expresses what
+Note how our `InvoiceBuilder` (which we show the code of soon) clearly expresses what
```
Comment (mauricioaniche) [inline chapters/pragmatic-testing/test-code-quality.md:651]: _code of_? Is that correct?

File: chapters/getting-started/test-automation.md (hunk 1 above)
Comment (MichalOkon) [reply chapters/getting-started/test-automation.md:200]: Done

**Comment by wouterpolet** [comment]
Hi @MichalOkon 
You probably added your commits to the `proofread-test-code-quality` branch, which causes one of my commits to be included in this PR.
We now merged #91 first and then merged this PR.
For next time try to make sure to branch from master (in your fork) and add your commits there, then the PR will only include your commits :slightly_smiling_face:

**Pull request merged**